front/src/config/i18n/en.json

```diff
@@ -223,6 +223,44 @@
       "saveError": "There was an error saving the camera.",
       "testConnectionError": "There was an error while getting the RTSP Flux. Are you sure the provided URL is right and accessible from Gladys instance?"
     },
+    "sonoff": {
+      "title": "Sonoff",
+      "deviceTab": "Devices",
+      "discoverTab": "MQTT discover",
+      "prepareDeviceDescr": "Or manually prepare future or not discoreved devices",
+      "discoverDeviceDescr": "Automatically scan MQTT devices",
+      "device": {
+        "title": "Sonoff devices in Gladys",
+        "search": "Search devices",
+        "newButton": "New",
+        "noDeviceFound": "No Sonoff device found. You can add devices with following actions :"
```

Comment:
"No Sonoff devices found. You can add devices with the following actions:"

Reply:
Done 😃